src/helpers/get-survey-answers.ts

```diff
@@ -0,0 +1,33 @@
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import {parseISO} from "date-fns";
+
+export default async function(props : SurveyAnswersQuery) : Promise<SurveyAnswer[]> {
+
+    const results = await getSurveyAnswers();
+    var sortedResults = (results).sort((a, b) => {
+            if (parseISO(a.date) > parseISO(b.date)) { return -1; }
+            if (parseISO(a.date) < parseISO(b.date)) { return 1; }
+            return 0;
+    });
+
+    return sortedResults; 
```

Comment:
Just a nitpick, but it was a bit confusing to me at first to have the return up here in the middle of the function.  I think it would read easier if the functions below were defined above (or as separate functions outside this function), with the return being the last thing that happens.  Just a personal preference, though.